cloudinit/config/schemas/schema-cloud-config-v1.json

```diff
@@ -577,6 +577,12 @@
         },
         "merge_type": {
           "$ref": "#/$defs/merge_definition"
+        },
+        "system_info": {
+          "type": "object",
+          "description": "System and/or distro specific settings. This is not intended to be overridden by user data or vendor data.",
+          "deprecated": true,
+          "deprecated_version": "24.2"
```

Comment:
Is the purpose of adding this to the schema, now that it's deprecated, to hopefully give user's warnings if they have the json schema loaded and attempt to use this "unsupported" config?

Reply:
+1 yes, basically. This should help avoid user confusion if they know the config works.

Ideally we would point them to the preferred option in a deprecation message, though this one is a little tricky since it's not really a 1:1 mapping - the users key is the preferred replacement for default_user, but other than that I'm not sure we really have a good story.